resources/views/aboutteam.blade.php

```diff
@@ -30,7 +30,7 @@
                             </div>
                             <div class="dlab-content">
                                 <div class="clearfix">
-                                    <h3 class="dlab-name"><a href="javascript:void(0);">John Doe</a></h3>
+                                    <h3 class="dlab-name"><a href="javascript:void(0);">Manoj</a></h3>
                                     <span class="dlab-position">Senior Designer</span>
```

Comment:
Considering this is a test, when we update it to actual work, will have to update this with relevant info